.github/downstream.d/scapy.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+case "${1}" in
+    install)
+        git clone --depth=1 https://github.com/secdev/scapy
+        cd scapy
+        git rev-parse HEAD
+	pip install tox
+        ;;
+    run)
+        cd scapy
+        tox -qe cryptography
```

Comment:
Can you add a comment saying that this uses the local cryptography because that tox env has `sitepackages = true` which makes it use externally installed stuff?

Also, is that understanding correct 😂

Reply:
That's correct ! 😛